integration-tests/config.ts

```diff
@@ -56,19 +56,19 @@ interface FaucetConfig {
   faucetKey: {};
 }
 
-const idiazabalnetEphemeral = {
-  rpc: process.env['TEZOS_RPC_IDIAZABALNET'] || 'https://idiazabalnet.ecadinfra.com',
-  knownBaker: 'tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD',
-  knownContract: 'KT1UUS3M31GpFS4k5xQtp97oWGDhdzdT9mNJ',
-  knownBigMapContract: 'KT1VFM8dyKv2YTFdLBZm8seJKgFnn7RMrDtv',
-  knownTzip1216Contract: 'KT1PiiNe7icQHeaQ4Z8m27FVhYWQ2pPGMots',
-  protocol: Protocols.ProtoALpha,
+const ithacanetEphemeral = {
+  rpc: process.env['TEZOS_RPC_ITHACANET'] || 'https://ithacanet.ecadinfra.com/',
+  knownBaker: 'tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD', 
+  knownContract: 'KT1J3BCZt5o8CA1xgqbV9Z1iEgoxuS1RamTa',
+  knownBigMapContract: 'KT1VFM8dyKv2YTFdLBZm8seJKgFnn7RMrDtv', 
```

Comment:
KT1VFM8dyKv2YTFdLBZm8seJKgFnn7RMrDtv is deployed on Idiazabalnet not Ithacanet

Reply:
should be KT1VC9dVmYYHyUXEw1KU8Ms6BmtvsGJNarZy

Reply:
changed